File: exercise/java/week01/exercise/src/main/java/movie/Movie.java
package movie;

public class Movie {
    public String movieID;
    public String title;
    public String director;
    public int totalCopies;
    public int borrowedCopies;

    public Movie(String movieID, String title, String director, int totalCopies) {
        this.movieID = movieID;
        this.title = title;
        this.director = director;
        this.totalCopies = totalCopies;
        this.borrowedCopies = 0;
    }
}

File: exercise/c#/week01-code-analysis/ExerciseMovieStore/Movie.cs
namespace ExerciseMovieStore;

public class Movie(string movieId, string title, string director, int totalCopies, double unitPrice)
{
    public string MovieId { get; set; } = movieId;
    public string Title { get; set; } = title;
    public string Director { get; set; } = director;
    public double UnitPrice { get; set; } = unitPrice;
    public int TotalCopies { get; set; } = totalCopies;
    public int BorrowedCopies { get; set; } = 0;

    public bool CanSell()
    {
        return UnitPrice != 0d;
    }
}